server/performance/metrics.go

```diff
@@ -36,8 +41,12 @@ type Metrics struct {
 
 	AppHandlersTimeHistograms *prometheus.HistogramVec
 
-	StoreOpCounters            *prometheus.CounterVec
-	StoreMethodsTimeHistograms *prometheus.HistogramVec
+	StoreOpCounters                      *prometheus.CounterVec
+	StoreMethodsTimeHistograms           *prometheus.HistogramVec
+	StoreDBWriterConnectionsInUseGauge   prometheus.GaugeFunc
+	StoreDBWriterConnectionsIdleGauge    prometheus.GaugeFunc
+	StoreDBWriterConnectionsWaitCount    prometheus.CounterFunc
+	StoreDBWriterConnectionsWaitDuration prometheus.CounterFunc
```

Comment:
I guess I'm missing something -- these must created and inc/dec-ed somewhere else?

Reply:
Ah, those need to go away. I figured out halfway through that https://pkg.go.dev/github.com/prometheus/client_golang@v1.19.0/prometheus/collectors#NewDBStatsCollector existed. Thanks for catching it.